corehq/ex-submodules/casexml/apps/phone/data_providers/case/providers.py

```diff
@@ -8,5 +9,7 @@ class CasePayloadProvider(FullResponseDataProvider):
     """
 
     def get_response(self, restore_state):
+        if not self.async_task and restore_state.do_livequery:
```

Comment:
you probably know this already, but we're going to want this in async restore when this goes live